governance.md

```diff
@@ -1,149 +1,285 @@
+
+- [**API REVIEWER**]:
+  - Requirements
+    - tenure requirement TBD
+    - nomination process TBD
+    - initial members TBD
+    - have written and/or reviewed Kubernetes APIs
+    - familiar enough with design, requirements, mechanics, conventions, style,
+      scope, gotchas, etc. of the API to be in `kubernetes/pkg/api` and `kubernetes/pkg/apis`
+      OWNERS files `reviewers` lists
+      - TODO: decouple design review from the firehose of PR reviews
+  - Expectations
+    - review API changes and proposals in their functional area
+    - ensure Kubernetes has a consistent [API style](contributors/devel/api-conventions.md),
+      patterns, and philosophies
+    - guide development of new APIs
+  - Benefits
+    - TBD
+    - added to [`api-reviewers`](https://github.com/orgs/kubernetes/teams/api-reviewers) github team
+- **API APPROVER**:
+  - Requirements
+    - tenure requirement TBD
+    - nomination process TBD
+    - initial members TBD (historically bgrant0607, thockin, smarterclayton, erictune)
+    - designed and reviewed several APIs in the system
+    - familiar with the design, requirements, mechanics, conventions, style,
+      scope, gotchas, etc. of the API to be in `kubernetes/pkg/api` and `kubernetes/pkg/apis`
+      OWNERS files `approvers` lists
+      - TODO: decouple design review from the firehose of PR reviews
+    - ensure Kubernetes has a consistent [API style](contributors/devel/api-conventions.md),
+      patterns, and philosophies
+    - guide development of new APIs
+    - codify new patterns for novel problems
+  - Expectations
+    - review and approve API changes and proposals in their functional area
+  - Benefits
+    - TBD
+    - added to [`api-approvers`](https://github.com/orgs/kubernetes/teams/api-approvers) github team
 
 ## SIG roles
-- **SIG Participant**: active in one or more areas of the project; wide 
-  variety of roles are represented
-- **SIG Lead**: SIG organizer
+- **SIG PARTICIPANT**: active in one or more areas of the project; wide variety of roles are represented
+- **SIG LEAD**: SIG organizer 
+  - Expectations 
+    - ensure meetings occur, decisions are communicated
+    - facilitate planning for each release and ensure feature process is followed
+    - ensure other SIGs are able to provide input / feedback
+    - ensure SIG follows governance guidelines, below
 
 ## Management roles
-- **Team Lead**: tech lead or manager of some team at some company working on 
-  K8s; can influence priorities of their team members; pragmatically, 
-  probably want label/assignment powers
-- [**kubernetes-pm**](https://github.com/orgs/kubernetes/teams/kubernetes-pm): help to [manage and maintain the project](project-managers/README.md) in
-  ways other than just writing code (e.g. managing issues); should subscribe to kubernetes-pm@googlegroups.com
+- [**PM**](https://github.com/orgs/kubernetes/teams/kubernetes-pm): help to [manage and
```

Comment:
This team arose out of github limitations.

Please put these comments into the umbrella governance issue. This group exists and is in use now.